scep/scep.go

```diff
@@ -146,11 +146,18 @@ func WithLogger(logger log.Logger) Option {
 	}
 }
 
+func WithCaCerts(caCerts []*x509.Certificate) Option {
```

Comment:
Needs a description to to indicate that it changes the verify behavior.

Reply:
Done.